Studio-Lovelies/GG-JointJustice-Unity
# Various findings as part of today's meeting

## Summary
Recommend checking out each commit individually. Resolves a few things:
- Adds colored nameplates for Burgie and Laura (51fc67d) while relying on Dan's animations to not break the docs generator (583c027) to get the demo `.ink` script closer to the Google Docs version
- `&NARRATE` had an incorrect example with a parameter that wasn't required (6045a66)
- Dan stopped talking when using the "Angry talk" sprite even if there was text left; it now correctly loops until all text is displayed (9bf62c9)
- My IDE kept adding files, so I've added it to the `.gitignore` as no one else is using Rider (5527251)
- Ross' animation `SadNoHelmetTalking` was incorrectly named `SadTalkingNoHelmet` in the documentation (7e3d55b)
- Added pudding haired leaning+talking sprite for Dan to prevent an empty scene until the new sprites are ready (e679c68)

## Testing instructions
- Verify the Burgie and Laura nameplates show up when used
- Look at the newly generated docs for `&NARRATE`
- Make Dan speak a long line while using the `Angry` pose and verify that mouth movements only stop after all text is displayed
- Make Dan speak and lean and verify that an animation is shown
## Additional information

- `[ ]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - closes #444
